CHANGELOG.rst

```diff
@@ -6,6 +6,28 @@
 Synapse Changelog
 *****************
 
+v2.197.0 - 2025-02-05
+=====================
+
+Model Changes
+-------------
+- Added ``doc:resume`` and ``doc:requirement`` to the data model.
+  (`#4104 <https://github.com/vertexproject/synapse/pull/4104>`_)
+- See :ref:`userguide_model_v2_197_0` for more detailed model changes.
+
+Features and Enhancements
+-------------------------
+- Added Unicode word character support for the ``ou:org:alias`` property.
+  (`#4108 <https://github.com/vertexproject/synapse/pull/4108>`_)
```

Comment:
Did we want to leave this out since we're deprecating it?

Reply:
IMO we can remove it. This should have been a `model` change anyhow.

Reply:
```suggestion
```